app/database_etl/airtable_records_handler/airtable_records_getter.py

```diff
@@ -84,3 +91,47 @@ def get_all_records():
         }
         send_api_error_slack_notif(body, data, error=e, request_info=request_info, channel='#dev-logging-etl')
         return request_info
+
+def ingest_sample_frame_goi_filter_options():
+    url = "https://api.airtable.com/v0/appFfYIupTrVQ0HJx/Sample%20Frame%20GOI?fields=Order&fields=Name&sort%5B0%5D%5Bfield%5D=Order&fields=French%20Name"
+    headers = {'Authorization': 'Bearer {}'.format(AIRTABLE_API_KEY)}
+
+    # Make request and retrieve records in json format
+    r = requests.get(url, headers=headers)
+    data = r.json()
+
+    # Try to get records from data if the request was successful
+    try:
+        # Get sorted records
+        records = data["records"]
+        drop_specific_table_entries(datetime.now(), PopulationGroupOptions)
+
+        # Add records to table 
+        with db_session() as session:
+            records_to_add = [PopulationGroupOptions(id=uuid4(), 
+                                                    name=record['fields']['Name'], 
+                                                    french_name=record['fields']['French Name'] if 'French Name' in record['fields'] else None, 
```

Comment:
Are you sure we won't need `record['fields']['Order'] if 'Order' in record['fields'] else None` as well? I don't think order will be guaranteed to be in the table

Reply:
from my understanding, i thought order would be there since it's linked to the filter options right and we have automations running for updates?

Reply:
@annoyingpr who can we check this with? who wrote the automation - christian?

Reply:
Prannoy is right I double checked. Either way though I wonder if it's a good idea to be fail safe and consistent with null checks here.